src/app/Components/ArtworkRail/ArtworkRailCardMeta.tsx

```diff
@@ -61,6 +62,17 @@ export const ArtworkRailCardMeta: React.FC<ArtworkRailCardMetaProps> = ({
 }) => {
   const { trackEvent } = useTracking()
   const enableAuctionImprovementsSignals = useFeatureFlag("AREnableAuctionImprovementsSignals")
+  const enableNewSaveAndFollowOnArtworkCard = useFeatureFlag(
+    "AREnableNewSaveAndFollowOnArtworkCard"
+  )
+  const newSaveAndFollowOnArtworkCardExperiment = useExperimentVariant(
+    "onyx_artwork-card-save-and-follow-cta-redesign"
+  )
+
+  const { enableShowOldSaveCTA } = getExperimentVariant(
+    newSaveAndFollowOnArtworkCardExperiment.enabled,
+    newSaveAndFollowOnArtworkCardExperiment.variant
+  )
```

Comment:
fwiw, I think you can even simplify this more by moving `useExperiementVariant` and `useFeatureFlag` to the hook
```suggestion
  const {enableShowOldSaveCTA} = useArtworkSomethingExperiment()
```